### DISCUSSION TITLE
Stable Diffusion 2.0

### DISCUSSION THREAD

**Discussion by seijihariki** [zero01101/openOutpaint#46] [General]
Hey, did you guys see the news about Stable Diffusion 2.0? Some exciting news ahead, it seems! Nothing really to do with this repo (we use automatic1111's API, after all), but the new model seems quite good; also supports depth img2img

**Comment by Kalekki** [comment]
[Official release post](https://stability.ai/blog/stable-diffusion-v2-release)
[Inpainting model](https://huggingface.co/stabilityai/stable-diffusion-2-inpainting)
[SD-2.0 768x768 model](https://huggingface.co/stabilityai/stable-diffusion-2)
[SD-2.0 512x512 model](https://huggingface.co/stabilityai/stable-diffusion-2-base)

I glanced through the results, and they indeed seem really good, not sure yet if the hardware requirements changed (or if so, by how much), but considering the models are >5gb they probably did.

Also seems that they now have the watermarking built-in to the model, so probably cant be turned off? I wonder if you can affect it with some model merging magic..

Training data is filtered for NSFW, which I completely understand considering the pressure they've amassed.
Celebrities and artists claimed to also be filtered out, which seems like a massive negative.

**Comment by zero01101** [comment]
god the samples look _fantastic_ no doubt; i guarantee 10 minutes won't pass after the ability to train against this model before there's at least a half-dozen NSFW models being shared around ;) 

yeah, if they did actually completely remove celebrities that's kind of a regression i think; i totally get the privacy and potential libel concerns but i mean the genie's kind of already out of the bottle... 

i _was_ a bit curious about how current SD could make AstroTurf Lawn Care™ and i'm certainly not disappointed :D

![image](https://user-images.githubusercontent.com/1649724/203812539-2ba32e2f-7a4c-49d2-a8fb-48dbd88f8c78.png)

**Comment by zero01101** [comment]
[here](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/ce6911158b5b2f9cf79b405a1f368f875492044d) we [go](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features)

--edit--

first things first: openOutpaint works just fine* with the 768-v sd2.0 model just as A1111 does, so that's sure neato

* - just fine in the sense that it doesn't break, it REALLY doesn't work at all for out/inpainting yet lol

secondly ok fine this is definitely a pretty significant lawncare service upgrade 

![00000-1472085101](https://user-images.githubusercontent.com/1649724/204119903-63af8d5f-5085-4fb1-971a-fefeaf691670.png)
![00001-1472085102](https://user-images.githubusercontent.com/1649724/204119905-53f21e0b-fa29-41d8-9e9d-ca9aadbe23f5.png)
